spec/support/mr_client_helpers.rb

```diff
@@ -1,13 +1,13 @@
 # frozen_string_literal: true
 
-require 'sm/client'
+require 'medical_records/client'
 
 module MedicalRecords
   module ClientHelpers
-    TOKEN = 'GkuX2OZ4dCE=48xrH6ObGXZ45ZAg70LBahi7CjswZe8SZGKMUVFIU88='
+    TOKEN = 'eyJhbGciOiJSUzI1NiJ9.eyJzdWIiOiIxMTg5ODc5NSIsImF1ZCI6IjEwMyxWQS5nb3YgTWVkaWNhbCBSZWNvcmRzIiwibmJmIjoxNjg3N'
```

Comment:
@mmoyer-va I know this is just a spec file, but is this token a sensitive value?